nixos/modules/services/x11/desktop-managers/cde.nix

```diff
@@ -8,9 +8,26 @@ let
 in {
   options.services.xserver.desktopManager.cde = {
     enable = mkEnableOption "Common Desktop Environment";
+
+    extraPackages = mkOption {
```

Comment:
What is the motivation to have an extra option here? Could you not just use `environment.systemPackages` directly?

Reply:
@markuskowa Its more nicer to put CDE specific packages there then to put them under systemPackages. mate, xmonad, i3, exwm have this option too.